pylint/checkers/symilar.py

```diff
@@ -876,67 +876,34 @@ def register(linter: PyLinter) -> None:
     linter.register_checker(SimilaritiesChecker(linter))
 
 
-def usage(status: int = 0) -> NoReturn:
-    """Display command line usage information."""
-    print("finds copy pasted blocks in a set of files")
-    print()
-    print(
-        "Usage: symilar [-d|--duplicates min_duplicated_lines] \
-[-i|--ignore-comments] [--ignore-docstrings] [--ignore-imports] [--ignore-signatures] file1..."
-    )
-    sys.exit(status)
```

Comment:
If this behavior is intended to remain, you may want to consider a solution similar to what's outlined here: https://stackoverflow.com/questions/3636967/how-can-i-automatically-display-the-help-on-command-line-arguments-error . There's `parser.print_help()` that could be output in case of error.